gf/test/one_index_gf_test.cpp

```diff
@@ -163,6 +163,7 @@ TEST_F(OneIndexGFTest, Legendre) {
 TEST_F(OneIndexGFTest, DefaultConstructive)
 {
     gf_type gf_empty;
+#ifndef NDEBUG
```

Comment:
The same comment as https://github.com/ALPSCore/ALPSCore/pull/408/files#diff-f5308a091bc5c9a3c0d44b94de128649R256